test/WebJobs.Script.Tests.Integration/TestScripts/CSharp/ProxyHttpResponse/response.json

```diff
@@ -0,0 +1,98 @@
+﻿{
+  "Content": {
```

Comment:
might be easier to maintain this to simply create a JObject inline in the test and ToString it to get this text? That way we don't have to check this file in